src/components/ProductItem/ProductItem.tsx

```diff
@@ -200,6 +200,7 @@ export const ProductItem: FunctionComponent<ProductProps> = ({
         eventInfo: {
           ...dl.getState(),
           sku: product?.sku,
+          productView,
```

Comment:
Not sure if it's a good idea to pass the whole productView to the datalayer, or would be better just to extract the few field/attributes that we need

Reply:
Yes. I can extract only required attributes.

Reply:
productView data was replaced with specific attributes data